api/v1/tetragon/sensors.proto

```diff
@@ -38,6 +38,26 @@ message ListSensorsResponse {
 	repeated SensorStatus sensors = 1;
 }
 
+message ListTracingPoliciesRequest { }
+
+message TracingPolicyStatus {
+	// id is the id of the policy
+	uint64 id = 1;
+	// name is the name of the policy
+	string name = 2;
+	// namespace is the namespace of the policy (or empty of the policy is global)
+	string namespace = 3;
+	// info is additional information about the policy
+	string info = 4;
+	// sensors loaded in the scope of this policy
+	repeated string sensors = 5;
```

Comment:
I guess we have a consistent way on how these sensors are named right?

Reply:
Each sensor has its name, yes.